examples/using-styled-components/src/pages/index.js

```diff
@@ -16,8 +16,16 @@ const Title = styled.h1`
   color: palevioletred;
 `
 
+// Create an URL component that'll render an <a> tag with some styles
+const Url = styled.a`
```

Comment:
Perhaps we just call this a `Link`? Or `ExternalLink` for clarity?

(the [URL global](https://developer.mozilla.org/en-US/docs/Web/API/URL)--while capitalized differently--can also be slightly confusing here!)

Reply:
I think I will call it `ExternalLink` so that no one will confuse it with Gatsby's [Link](https://www.gatsbyjs.org/docs/gatsby-link/)